package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@bitfinex/lib-js-util-math": "git+https://github.com/bitfinexcom/lib-js-util-math.git",
     "bfx-api-node-core": "^1.5.5",
-    "bfx-api-node-models": "^1.4.1",
+    "bfx-api-node-models": "git+https://github.com/avsek477/bfx-api-node-models.git#add-visible-on-hit-hidden-orders",
```

Comment:
needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-models/pull/77) is merged.